files/en-us/learn/server-side/configuring_server_mime_types/index.html

```diff
@@ -109,14 +109,20 @@ <h2 id="How_to_set_up_your_server_to_send_the_correct_MIME_types">How to set up
 
 <ul>
  <li>If you're using the Apache web server, check the <em><strong>Media Types and Character Encodings</strong></em> section of <a href="/en-US/docs/Learn/Server-side/Apache_Configuration_htaccess">Apache Configuration: .htaccess</a> for examples of different document types and their corresponding MIME types.</li>
- <li>If you're using NGINX, look at the <a href="/en-US/NGINX_configuration_snippets">NGINX configuration snippets</a>. NGINX does not have a <code>.htaccess</code> equivalent tool, so all changes will go into the main configuration file.</li>
+ <li>If you're using NGINX, note that NGINX does not have a <code>.htaccess</code> equivalent tool, so all changes will go into the main configuration file.</li>
  <li>If you're using a server-side script or framework to generate content, the way to indicate the content type will depend on the tool you're using. Check the framework or library's documentation.</li>
 </ul>
 
+<p>Regardless of what server system you use, the effect you need to achieve is to set a response header with the name {{httpheader("Content-Type")}}, followed by a colon and space, followed by a MIME type. For example, in a PHP environment, you’d need to set the response header for PDF resources like this:
```

Comment:
I think this is not 100% accurate. You can set it up at the server level as described above, or you can set it up at the php level with `header()`. The sentence sounds like you need to do it at the php level, in addition to the web server level
```suggestion
<p>Regardless of what server system you use, the effect you need to achieve is to set a response header with the name {{httpheader("Content-Type")}}, followed by a colon and space, followed by a MIME type. High level environments often allow such headers to be set when generating the page. For example, in a PHP environment, you could set the response header for PDF resources like this:
```